### PR TITLE
pep440: unify on one package

### PR DESCRIPTION
**Pull request by hdonnay**
This moves all pep440 usage over to the local package instead of being split across two packages.

### CODE REVIEW

File: pkg/pep440/range.go [hunk 1]
```diff
@@ -6,39 +6,43 @@ import (
 	"unicode"
 )
 
+//go:generate stringer -linecomment -type op
```
Comment (crozzy) [inline pkg/pep440/range.go:9]: https://github.com/quay/claircore/blob/b10fa7900054a953584c68777bff9c919406af00/pkg/pep440/range.go#L58 could maybe simplify this now that it's been stringered.

File: python/matcher.go [hunk 2]
```diff
@@ -39,16 +39,20 @@ func (*Matcher) Vulnerable(ctx context.Context, record *claircore.IndexRecord, v
 
 	v, err := pep440.Parse(record.Package.Version)
 	if err != nil {
+		zlog.Info(ctx).
```
Comment (crozzy) [inline python/matcher.go:42]: Thoughts on these being `warn`s?

**Review by crozzy — COMMENTED** [review]
Looks good, couple of comments

File: python/matcher.go (hunk 2 above)
Comment (hdonnay) [reply python/matcher.go:42]: Yeah, okay. If these are this far into the system without anything noticing the versions are weird, that's more of a problem.

**Review by crozzy — APPROVED** [review]

**Pull request merged**